shared/elements/gaia_grid/js/items/group.js

```diff
@@ -344,6 +353,12 @@
         } else {
           // If we're not dragging, save the collapsed state
           window.dispatchEvent(new CustomEvent('gaiagrid-saveitems'));
+
+          // Request attention so that we're as visible as we can be after
+          // expanding/collapsing
+          setTimeout(() => {
```

Comment:
Can we use a rAF here? I understand we are inside a longer timeout to avoid coalescing with a previous action, but maybe rAF would work now since we've already avoided the coalesce?